quora-service/src/main/java/com/upgrad/quora/service/entity/UserAuthTokenEntity.java

```diff
@@ -0,0 +1,170 @@
+package com.upgrad.quora.service.entity;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.apache.commons.lang3.builder.ToStringStyle;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.io.Serializable;
+import java.time.ZonedDateTime;
+
+@Entity
+@Table(name = "user_auth_tokens", schema = "qoura")
+@NamedQueries({
+        @NamedQuery(name = "userAuthTokenByAccessToken", query = "select ut from UserAuthTokenEntity ut where ut.accessToken = :accessToken ")
+})
+public class UserAuthTokenEntity implements Serializable {
+
+
+    @Id
+    @Column(name = "ID")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+
+    @ManyToOne
+    @JoinColumn(name = "USER_ID")
+    private UserEntity user;
+
+    @Column(name = "ACCESS_TOKEN")
+    @NotNull
+    @Size(max = 500)
+    private String accessToken;
+
+    @Column(name = "LOGIN_AT")
+    @NotNull
+    private ZonedDateTime loginAt;
+
+    @Column(name = "EXPIRES_AT")
+    @NotNull
+    private ZonedDateTime expiresAt;
+
+    @Column(name = "LOGOUT_AT")
+    private ZonedDateTime logoutAt;
+
+
+    @Version
+    @Column(name = "VERSION", length = 19, nullable = false)
+    private Long version;
+
+
+    @Column(name = "CREATED_BY")
+    @NotNull
+    private String createdBy;
+
+
+    @Column(name = "CREATED_AT")
+    @NotNull
+    private ZonedDateTime createdAt;
```

Comment:
This is not as per requirement document. Please make the changes accordingly.
Additional fields like createdAt and version are not required.

Reply:
Went through the database document and fixed the issue in the latest commit. Thanks!

Reply:
Ok. Good Job !!!